### PR TITLE
FEATURE: Link Neos logo in modules to primary module

### LINKED ISSUE #4799: FEATURE: Use the Neos logo in the top bar as "home button"
### Is there an existing issue for this topic?

- [X] I have searched the existing issues

### Description

I often want to directly jump to the content module which is the default first available `preferredStartModules` setting.

So instead of clicking to open the module dropdown then selection the content module etc. one click on the Neos logo would direct me there instead.

If the `preferredStartModules` is configured differently or users are restricted to another module, for them the "home" would just be that next module then and hopefully not be confusing.

### Possible Solution

Link the logo to the first available (defined by policies) entry in `preferredStartModules`.

### PR DESCRIPTION
**Pull request by Sebobo**
In the default configuration this links to the content module, or to their user settings if they don’t have access to the content module.

Resolves: #4799

### CODE REVIEW

File: Neos.Neos/Resources/Private/Partials/Backend/Branding.html [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
-			<path class="cls-2" d="M88.83 0L68.12 15.1v31.57l20.71 29.28V0zM88.83 112.57L9.22 0 0 6.74v117.71l20.71-15.1V51.06l51.78 73.39h22.65l16.34-11.88H88.83z"/>
-			<path class="cls-1" d="M20.71 51.06v58.29L0 124.45h22.65l20.71-15.1V83.17L20.71 51.06zM88.83 75.95V0h22.65v112.57H88.83L9.22 0h25.89l53.72 75.95z"/>
+	<a href="{primaryModuleUri}" title="{neos:backend.translate(id: 'branding.link.title', source: 'Modules')}">
+		<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 453.54 124.45" class="neos-branding-logo">
```
Comment (jonnitto) [inline Neos.Neos/Resources/Private/Partials/Backend/Branding.html:4]: I think we could remove some markup from the Neos logo, the `<g>` tags are not needed:

```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 453.54 124.45" class="neos-branding-logo">
    <defs>
        <style>
        .white{fill:#fff}.blue{fill:#009fe3}
        </style>
    </defs>
    <path class="white" d="M410.69 86l4.95-5.54c5.5 4.94 10.61 7.54 18.11 7.54 6.7 0 11.49-3.4 11.49-8.47 0-4.51-2.55-6.89-13.49-9.66-12.69-3.25-19.07-7.6-19.07-16.47 0-9.19 8.46-15.52 19.31-15.52 8.06 0 14 2.62 19.71 7.13l-4.63 5.94c-5.27-4.12-9.74-5.86-15.16-5.86-6.15 0-10.86 3-10.86 7.76 0 4.59 3 7.13 14.29 9.9 12.21 2.93 18.2 7.76 18.2 16.31 0 10-8.62 16.16-19.87 16.16-8.94 0-16.36-3.22-22.98-9.22zM327.77 66.55c0-15.68 12.13-28.67 29-28.67s29 13 29 28.67-12.13 28.67-29 28.67-29-12.99-29-28.67zm49.72 0c0-12.12-8.54-21.3-20.67-21.3s-20.67 9.19-20.67 21.3 8.54 21.3 20.67 21.3 20.67-9.19 20.67-21.3zM302.55 38.83V46h-32.72v16.67H299V70h-29.17v17.06h33.12v7.21h-41.18V38.83zM191.07 38.83l32.48 40.94V38.83h7.9v55.44h-6.62l-33.37-42.06v42.06h-7.9V38.83z"/>
    <path class="blue" d="M88.83 0L68.12 15.1v31.57l20.71 29.28V0zM88.83 112.57L9.22 0 0 6.74v117.71l20.71-15.1V51.06l51.78 73.39h22.65l16.34-11.88H88.83z"/>
    <path class="white" d="M20.71 51.06v58.29L0 124.45h22.65l20.71-15.1V83.17L20.71 51.06zM88.83 75.95V0h22.65v112.57H88.83L9.22 0h25.89l53.72 75.95z"/>
</svg>
```

**Review by crydotsnake — APPROVED** [review]
Looks, and works great for me!

**Comment by dlubitz** [comment]
For some reaon the pipeline fails all the time in this PR

File: Neos.Neos/Resources/Private/Partials/Backend/Branding.html (hunk 1 above)
Comment (Sebobo) [reply Neos.Neos/Resources/Private/Partials/Backend/Branding.html:4]: Removed the g and inlined the style

**Review by ahaeslich — APPROVED** [review]

**Pull request merged**